feature-tests/utils/utility.ts

```diff
@@ -22,3 +24,24 @@ export interface CurrentTimeDescriptor {
   milliseconds: number;
   seconds: number;
 }
+
+export type inputObjectForUpdatingItem = {
+  updatedAt: number;
+  appliedAt: number;
+  sentAt: number;
+  intervention: string;
+  blocked: boolean;
+  suspended: boolean;
+  resetPassword: boolean;
+  reproveIdentity: boolean;
+  history: string;
+  auditLevel: string;
+  isAccountDeleted?: boolean;
+  reprovedIdentityAt?: number;
+  resetPasswordAt?: number;
+  accountDeletedAt?: number;
+};
+
+export interface UserInformationFromTable {
```

Comment:
hey Liam - I do not think this is right actually... Or at leat this is not the level that the interface should be at. This should describe a JSON object (like the one above) with the expected fields. The unmarshall call shouldn't return a Record type